src/workingtitle-vcockpits-instruments-navsystems-g3000/html_ui/WTg3000/Avionics/Shared/PFD/Main/G3x5_PFD.js

```diff
@@ -554,10 +557,18 @@ class AS3000_PFD_MainElement extends NavSystemElement {
 }
 
 class AS3000_PFD_Attitude extends PFD_Attitude {
-    constructor(instrumentID) {
+    /**
+     * @param {String} instrumentID
```

Comment:
(if you're using typescript for type information) you should use lowercase `string` for strings, as `String` would be a boxed object created with `new String()`; same with `Boolean` (and `Symbol` if it had a constructor)

Reply:
These changes are coming from Asobo